backend/src/nodes/nodes/stable_diffusion/__init__.py

```diff
@@ -0,0 +1,9 @@
+from ...category import Category
+
+category = Category(
+    name="Stable Diffusion",
+    description="Nodes for using Stable Diffusion",
+    icon="PyTorch",
+    color="#DD6B20",
```

Comment:
IMO these should probably be a different icon & color (like the webui ones). While they are still PyTorch, I think it's separate enough where it makes sense to make it unique. The webui nodes I made purple, which coincidentally also happens to be the same color as [pytorch lightning](https://lightning.ai/docs/pytorch/stable/). Might at least be good to use the lightning logo for the load checkpoint node.

Reply:
Yeah, I just used the color and icon of whatever node I copied to get started with.    I changed the color to match the external SD one, and changed the icons to something suitably vague.  Hard to find good icons to represent the things these nodes do, at least without designing our own icons.  I tried to use the `SiPytorchLightling` icon from here https://react-icons.github.io/react-icons/search?q=lightning, but it doesn't seem to be part of the icon library we're using.

Reply:
We probably need to update the package

Reply:
Thank you!